pkg/test/deployment/helm.go

```diff
@@ -144,13 +144,13 @@ func HelmTemplate(deploymentName, namespace, chartDir, workDir, valuesFile strin
 	}
 
 	// Adding cni dependency as a workaround for now.
-	if _, err := exec(fmt.Sprintf("helm --home %s repo add istio.io %s",
-		helmRepoDir, "https://storage.googleapis.com/istio-prerelease/daily-build/master-latest-daily/charts")); err != nil {
-		return "", err
-	}
+	// if _, err := exec(fmt.Sprintf("helm --home %s repo add istio.io %s",
```

Comment:
Simply remove this, or add a TODO explaining why the commented out stuff should stay?